boards/arm/holyiot_yj16019/holyiot_yj16019.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: holyiot_yj16019
+name: Holyiot YJ-16019
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - ble
+  - ieee802154
+  - pwm
+  - watchdog
+  - counter
```

Comment:
probably good to add flash/ram size info in to holyiot_yj16019.yaml

Reply:
Added, thanks.